tutorials/XGI in 15 minutes.ipynb

```diff
@@ -0,0 +1,679 @@
+{
```

Comment:
<p>Remove ' for example" ', at the beginning of the sentence</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>